taskt/Core/Automation/Commands/Error Handling Commands/GetExceptionMessageCommand.cs

```diff
@@ -1,57 +1,60 @@
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Error Handling Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to store an exception error message in a variable.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to save an exception error message")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("Error Handling Commands")]
+    [Description("This command allows you to store an exception error message in a variable.")]
     public class GetExceptionMessageCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please define where the error text should be stored")]
-        [Attributes.PropertyAttributes.InputSpecification("Select or provide a variable from the variable list")]
-        [Attributes.PropertyAttributes.SampleUsage("**vSomeVariable**")]
-        [Attributes.PropertyAttributes.Remarks("If you have enabled the setting **Create Missing Variables at Runtime** then you are not required to pre-define your variables, however, it is highly recommended.")]
-        public string v_userVariableName { get; set; }
+        [PropertyDescription("Output String Variable")]
+        [InputSpecification("Choose a variable to take the output error string.")]
+        [SampleUsage("vSomeVariable")]
+        [Remarks("If you have enabled the setting **Create Missing Variables at Runtime** then you are not required " +
+                 "to pre-define your variables, however, it is highly recommended.")]
+        public string v_OutputUserVariableName { get; set; }
 
         public GetExceptionMessageCommand()
         {
-            this.CommandName = "GetExceptionMessageCommand";
-            this.SelectionName = "Get Exception Message";
-            this.CommandEnabled = true;
-            this.CustomRendering = true;
+            CommandName = "GetExceptionMessageCommand";
+            SelectionName = "Get Exception Message";
+            CommandEnabled = true;
+            CustomRendering = true;
         }
+
         public override void RunCommand(object sender)
         {
-            var engine = (Core.Automation.Engine.AutomationEngineInstance)sender;
+            var engine = (AutomationEngineInstance)sender;
             var error = engine.ErrorsOccured.OrderByDescending(x => x.LineNumber).FirstOrDefault();
-            var errorMessage = engine.FileName + " Line " + error.LineNumber.ToString() + "Error: " + error.ErrorMessage;
-            errorMessage.StoreInUserVariable(sender, v_userVariableName);
-
+            var errorMessage = engine.FileName + " Line " + error.LineNumber + "Error: " + error.ErrorMessage;
```

Comment:
can you change this to ", Error: ". I forgot to do this when I made this and it's been bothering me ever since.

Reply:
Done also, thanks for catching